dev-util/emscripten/emscripten-1.34.11.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1
+
+DESCRIPTION=" Emscripten: An LLVM-to-JavaScript Compiler"
+HOMEPAGE="http://emscripten.org/"
+SRC_URI="https://github.com/kripken/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="UoI-NCSA"
+
+MYDEPEND="${PYTHON_DEPS}
+	=dev-util/emscripten-fastcomp-1.34.11
+	net-libs/nodejs"
+DEPEND="${MYDEPEND}"
+RDEPEND="${MYDEPEND}"
+
+DEST="/usr/share/"
+TEST="${WORKDIR}/test/"
+
+src_prepare() {
+	cp "${FILESDIR}/emscripten.config" "${S}/"
+	cp "${FILESDIR}/99emscripten" "${S}/"
+}
+
+src_test() {
```

Comment:
No, because the test flag is a "FEATURES" https://devmanual.gentoo.org/general-concepts/features/index.html .